src/Take.Blip.Builder.UnitTests/InputExpirationHandlerTests.cs

```diff
@@ -174,6 +175,112 @@ await Scheduler
                     Arg.Any<Node>(),
                     Arg.Is<CancellationToken>(c => !c.IsCancellationRequested));
         }
+
+        [Fact]
+        public async Task WhenUserSendsMessage_WithoutInputExpiration()
+        {
+            // Arrange
+            Message.Content = new PlainText() { Text = "Teste" };
+
+            var state = new State
+            {
+                Id = "ping",
+                Input = new Builder.Models.Input()
+                {
+                    Expiration = TimeSpan.FromMinutes(1)
+                }
+            };
+
+
+            // Act
+            await InputHandler.OnFlowPreProcessingAsync(state, Message, null, default(CancellationToken));
+
+            // Assert
+            await _inputExpirationCount
+                .Received(1)
+                .TryRemoveAsync(Message);
+        }
```

Comment:
already exists a test fo it called "WhenUserSendMessageCancelInputExpirationScheduled".

move the "_inputExpirationCount" to WhenUserSendMessageCancelInputExpirationScheduled